src/index.js

```diff
@@ -1,43 +1,38 @@
 import './style.css';
+import Todos from './todos.js';
 
 const list = document.getElementById('list');
+const input = document.getElementById('input');
 
-const TodoList = [
-  {
-    index: 1,
-    description: 'DO dishes',
-    completed: false,
-  },
+const MyTodos = new Todos();
 
-  {
-    index: 3,
-    description: 'clean house',
-    completed: false,
-  },
-
-  {
-    index: 2,
-    description: 'go to supermarket',
-    completed: false,
-  },
-];
-
-function addTodo() {
-  TodoList.sort((a, b) => a.index - b.index).forEach((todo) => {
-    const item = `
-  <li class="item">
-            <div>
-              <input type="checkbox" id=${todo.index} name="checkbox" />
-              <span>${todo.description}</span>
-            </div>
-            <i class="fa-regular fa-ellipsis-vertical"></i>
-          </li>
-  `;
-
-    list.insertAdjacentHTML('beforeend', item);
-  });
-}
+let id = MyTodos.TodoList.length;
 
 window.onload = () => {
-  addTodo();
+  MyTodos.renderTodos();
 };
+
+document.addEventListener('keydown', (event) => {
+  if (event.code === 'Enter') {
+    const toDo = input.value;
+    if (toDo) {
```

Comment:
I would recommend that you listen to the particular element that takes the inputs rather than the whole document. This will help to avoid some behaviors you might not expect. For example, when a user presses enter and the input element is not in focus, the item would get saved.

Also, there seems to be a console.log statement, kindly check and remove it.

![image](https://user-images.githubusercontent.com/20554583/199873351-430dbe89-dfc9-423c-a525-f1ba74db0044.png)  
![image](https://user-images.githubusercontent.com/20554583/199873370-60d86e16-8834-4d49-a90a-3829ee7115a2.png)

Reply:
Thank you for the recommendation. I will make sure to update the code to reflect this and avoid any unexpected behaviors.